packages/pluggableWidgets/pie-doughnut-chart-native/src/PieDoughnutChart.editorConfig.ts

```diff
@@ -0,0 +1,32 @@
+import { StructurePreviewProps } from "@mendix/piw-utils-internal";
+
+import { PieDoughnutChartPreviewProps } from "../typings/PieDoughnutChartProps";
+import PieWithLabels from "./assets/pie-w-labels.svg";
+import PieWithoutLabels from "./assets/pie-wo-labels.svg";
+import DoughnutWithLabels from "./assets/doughnut-w-labels.svg";
+import DoughnutWithoutLabels from "./assets/doughnut-wo-labels.svg";
+
+export function getPreview(values: PieDoughnutChartPreviewProps): StructurePreviewProps {
+    if (values.presentation === "pie") {
+        if (values.showLabels) {
+            return result(PieWithLabels);
+        }
+
+        return result(PieWithoutLabels);
+    }
+
+    if (values.showLabels) {
+        return result(DoughnutWithLabels);
+    }
+
+    return result(DoughnutWithoutLabels);
```

Comment:
I think you can shorten this a bit. Something like;
```js
return values.presentation === "pie" ? 
    result(values.showLabels ? PieWithLabels : PieWithoutLabels) : 
    result(values.showLabels ? DoughnutWithLabels : DoughnutWithoutLabels)
```

Reply:
Great, Wes! 👏